ddl/error.go

```diff
@@ -53,15 +53,16 @@ var (
 	errIncorrectPrefixKey                     = dbterror.ClassDDL.NewStd(mysql.ErrWrongSubKey)
 	errTooLongKey                             = dbterror.ClassDDL.NewStd(mysql.ErrTooLongKey)
 	errKeyColumnDoesNotExits                  = dbterror.ClassDDL.NewStd(mysql.ErrKeyColumnDoesNotExits)
-	errUnknownTypeLength                      = dbterror.ClassDDL.NewStd(mysql.ErrUnknownTypeLength)
-	errInvalidDDLJobVersion                   = dbterror.ClassDDL.NewStd(mysql.ErrInvalidDDLJobVersion)
-	errInvalidUseOfNull                       = dbterror.ClassDDL.NewStd(mysql.ErrInvalidUseOfNull)
-	errTooManyFields                          = dbterror.ClassDDL.NewStd(mysql.ErrTooManyFields)
-	errTooManyKeys                            = dbterror.ClassDDL.NewStd(mysql.ErrTooManyKeys)
-	errInvalidSplitRegionRanges               = dbterror.ClassDDL.NewStd(mysql.ErrInvalidSplitRegionRanges)
-	errReorgPanic                             = dbterror.ClassDDL.NewStd(mysql.ErrReorgPanic)
-	errFkColumnCannotDrop                     = dbterror.ClassDDL.NewStd(mysql.ErrFkColumnCannotDrop)
-	errFKIncompatibleColumns                  = dbterror.ClassDDL.NewStd(mysql.ErrFKIncompatibleColumns)
+	// nolint:unused
+	errUnknownTypeLength        = dbterror.ClassDDL.NewStd(mysql.ErrUnknownTypeLength)
+	errInvalidDDLJobVersion     = dbterror.ClassDDL.NewStd(mysql.ErrInvalidDDLJobVersion)
+	errInvalidUseOfNull         = dbterror.ClassDDL.NewStd(mysql.ErrInvalidUseOfNull)
+	errTooManyFields            = dbterror.ClassDDL.NewStd(mysql.ErrTooManyFields)
+	errTooManyKeys              = dbterror.ClassDDL.NewStd(mysql.ErrTooManyKeys)
+	errInvalidSplitRegionRanges = dbterror.ClassDDL.NewStd(mysql.ErrInvalidSplitRegionRanges)
+	errReorgPanic               = dbterror.ClassDDL.NewStd(mysql.ErrReorgPanic)
+	errFkColumnCannotDrop       = dbterror.ClassDDL.NewStd(mysql.ErrFkColumnCannotDrop)
+	errFKIncompatibleColumns    = dbterror.ClassDDL.NewStd(mysql.ErrFKIncompatibleColumns)
```

Comment:
Could we remove these variables?

Reply:
I'm not sure, the error convert from mysql. should we remove it?

Reply:
+1 We can add them on demand.